src/indicators/algorithms/exponentialMovingAverage.js

```diff
@@ -0,0 +1,49 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.indicators.algorithms.exponentialMovingAverage = function() {
+
+        var days = 9,
+            value = fc.utilities.fn.identity;
+
+        function undefinedArrayOfLength(length) {
+            return Array.apply(undefined, new Array(length));
+        }
+
+        var exponentialMovingAverage = function(data) {
```

Comment:
I wonder if this could be rewritten more in the style of the sliding window function? My preference is based on the top level ```data.map``` function making it obvious that the resulting array will always be of the same length.